index.html

```diff
@@ -395,6 +396,35 @@
       </ul>
     </section>
   </section>
+  <section id="privacy" class="informative">
+    <h2>Privacy and Security</h2>
+    <p>Preload is a declarative fetch primitive that initiates early fetch of
+    resources and separates fetching from resource execution. In effect, it is
+    conceptually similar to initiating a scripted fetch for a resource, but
+    with additional constraints and benefits:</p>
+    <ul>
+      <li>The resource declaration is visible via HTTP headers or HTML markup,
+      which allows the user agent to initiate the fetch earlier, and without
+      blocking on JavaScript execution.</li>
+      <li>The response is not executed and there is no callback to retrieve its
+      payload. The response is retrieved and executed by initiating another
+      fetch (via script, DOM element, etc) that matches the preloaded
+      response.</li>
+      <li>The developer can specify the target context that will consume the
+      response via `as` attribute, which allows the user agent to enforce the
+      relevant CSP policies when initiating the preload fetch. If `as` is
+      omitted, preload defaults to same security and privacy processing as a
+      call to `fetch()` - i.e. subject to `connect-src`.</li>
+    </ul>
+    <p>The site authors are encouraged to take the necessary precautions and
```

Comment:
I'm inclined to omit this for Preload, exactly because the same can be said for any resource fetch that the page is already making -- preload is just initiating the request earlier.

That said, we do highlight this within Resource Hints.. and I think it does make sense to do so there as we're talking about fetching resources that are required by some other page.